DECISION-MAKING.md

```diff
@@ -126,7 +140,7 @@ Repository:
 ## IAM Infrastructure
 
 - Owner (accountable, decision maker): TBD
-- Peers: @danielh (infra) @adelbarrio (monitoring&alerting) @akrug @johngian
+- Peers: @danielh (infra) @adelbarrio (monitoring&alerting) @akrug @fiji-flo
```

Comment:
@gdestuynder do we have an update for this?

Reply:
ive a meeting on this by EOD today